SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme

```diff
@@ -88,6 +88,13 @@
             ReferencedContainer = "container:SwiftGen.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
```

Comment:
Instead of adding an env. variable here (which is quite hidden), I'd modify the `Config.init` to accept an environment dictionary (defaulting to `ProcessInfo.processInfo.environment`). That way, in the test below you can provide the necessary values in the test itself.